source/LogbookPanel.h

```diff
@@ -58,8 +59,8 @@ class LogbookPanel : public Panel {
 	// Current month being displayed:
 	Date selectedDate;
 	std::string selectedName;
-	std::multimap<Date, std::string>::const_iterator begin;
-	std::multimap<Date, std::string>::const_iterator end;
+	std::multimap<Date, std::vector<MediaNode>>::const_iterator begin;
+	std::multimap<Date, std::vector<MediaNode>>::const_iterator end;
```

Comment:
I think you're assigning `std::map` iterators to these, not `std::multimap` anymore.